zkevm-circuits/src/util/word.rs

```diff
@@ -359,6 +391,13 @@ impl<F: Field> Word<Value<F>> {
     }
 }
 
+impl Word<Column<Advice>> {
+    /// Query advice of Word of columns advice
+    pub fn query<F: Field>(&self, meta: &mut VirtualCells<F>, at: Rotation) -> Word<Expression<F>> {
```

Comment:
I think `query_advice` is more self-explanation than `query`

Reply:
Done